keyboards/jcpm2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,160 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Define the layers
+enum layer_names {
+    _LAYER0,
+    _LAYER0_MOD,
+    _LAYER1,
+    _LAYER2,
+    _LAYER3 //unused
+};
+
+const uint8_t UNDERGLOW = 60;
```

Comment:
Or just substitute 60 in directly...
```suggestion
#define UNDERGLOW 60
```